configuration/configuration.go

```diff
@@ -172,6 +172,24 @@ type Configuration struct {
 			TrustKey string `yaml:"signingkeyfile,omitempty"`
 		} `yaml:"schema1,omitempty"`
 	} `yaml:"compatibility,omitempty"`
+
+	// Validation configures validation options for the registry.
+	Validation struct {
+		// Disabled disables all other options in this section.
+		Disabled bool `yaml:"disabled,omitempty"`
```

Comment:
By default, this should be "disabled". The zero-value of `Disabled`, would be `false`, which would mean `Enabled`. This field should be called `Enabled` and users should be required to set it to true to enable the validation.

I think, by default, we can refuse manifests with anything in the url field.